generators/component/index.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+var yosay = require('yosay');
+var _ = require('lodash');
+var helper = require('../../helper');
+var fs = require('fs');
+
+module.exports = yeoman.Base.extend({
+  prompting: function() {
+    this.log(yosay(
+      chalk.red('generator-ng-flash') + ' component generator!'
+    ));
+
+    var prompts = [{
+        type: 'input',
+        name: 'name',
+        message: 'Component name?',
+        filter: (name) => {
+          return _.camelCase(name);
+        }
+      }];
+
+    return this.prompt(prompts).then(function (props) {
+      this.props = props;
+    }.bind(this));
+  },
+
+  writing: function() {
+    helper.appendComponent(this.destinationRoot(), this.props.name);
+
+    fs.mkdirSync(this.destinationRoot() + '/' + this.props.name);
+
+    var files = helper.files(__dirname + '/templates');
+    _.each(files, (file) => {
+      var path = file.split('templates/')[1];
```

Comment:
'template' cover this into constant. `Magic words`

Better if convert it into function